app/src/test/java/org/mozilla/fenix/ext/BookmarkNodeTest.kt

```diff
@@ -17,12 +17,15 @@ import mozilla.components.concept.storage.BookmarkNodeType
 @Config(application = TestApplication::class)
 class BookmarkNodeTest {
 
+    val bookMarkChild1 = newBookmarkNode("Child 1", 1, null)
+    val bookMarkChild2 = newBookmarkNode("Child 2", 2, null)
+    val bookMarkChild3 = newBookmarkNode("Child 3", 3, null)
+    val listOfAllChildren = listOf(bookMarkChild1, bookMarkChild2)
+    val setOfAllChildren = setOf(bookMarkChild1, bookMarkChild2)
```

Comment:
Since the set is only used in one test, let's name the list `allChildren` and use `allChildren.toSet()`